docs/modeling-your-data/modeling-your-data-with-dbt/dbt-quickstart/unified/index.md

```diff
@@ -0,0 +1,167 @@
+
+- [IAB enrichment](/docs/enriching-your-data/available-enrichments/iab-enrichment/index.md)
+- [UA Parser enrichment](/docs/enriching-your-data/available-enrichments/ua-parser-enrichment/index.md)
+- [YAUAA enrichment](/docs/enriching-your-data/available-enrichments/yauaa-enrichment/index.md)
+- Browser context
+- Mobile context
+- Geolocation context
+- App context
+- Screen context
+- Deep Link context
+- App Error context
+- Core Web Vitals
+- Consent (Preferences & cmp visible)
+
+By default these are **all disabled** in the unified package. Assuming you have the enrichments turned on in your Snowplow pipeline, to enable the contexts within the package please add the following to your `dbt_project.yml` file:
+
+```yml title="dbt_project.yml"
+vars:
+  snowplow_unified:
+    snowplow__enable_iab: true
+    snowplow__enable_ua: true
+    snowplow__enable_yauaa: true
+    snowplow__enable_browser_context: false
+    snowplow__enable_mobile_context: false
+    snowplow__enable_geolocation_context: false
+    snowplow__enable_application_context: false
+    snowplow__enable_screen_context: false
+    snowplow__enable_deep_link_context: false
+    snowplow__enable_consent: false
+    snowplow__enable_cwv: false
+    snowplow__enable_app_errors: false
+```
+
+### 5. Filter your data set
+
+You can specify both `start_date` at which to start processing events and the `app_id`'s to filter for. By default the `start_date` is set to `2020-01-01` and all `app_id`'s are selected. To change this please add the following to your `dbt_project.yml` file:
+
+```yml title="dbt_project.yml"
+vars:
+  snowplow_unified:
+    snowplow__start_date: 'yyyy-mm-dd'
+    snowplow__app_id: ['my_app_1','my_app_2']
+```
+
+
+### 6. Verify page ping variables
+
+The unified package processes page ping events to calculate web page engagement times. If your [tracker configuration](/docs/collecting-data/collecting-from-own-applications/javascript-trackers/web-tracker/tracking-events/index.md#activity-tracking-page-pings) for `min_visit_length` (default 5) and `heartbeat` (default 10) differs from the defaults provided in this package, you can override by adding to your `dbt_project.yml`:
+
+```yml title="dbt_project.yml"
+vars:
+  snowplow_unified:
+    snowplow__min_visit_length: 5 # Default value
+    snowplow__heartbeat: 10 # Default value
+```
+
+### 7. Additional vendor specific configuration
+
+:::info BigQuery Only
+Verify which column your events table is partitioned on. It will likely be partitioned on `collector_tstamp` or `derived_tstamp`. If it is partitioned on `collector_tstamp` you should set `snowplow__derived_tstamp_partitioned` to `false`. This will ensure only the `collector_tstamp` column is used for partition pruning when querying the events table:
+
+```yml title="dbt_project.yml"
+vars:
+  snowplow_unified:
+    snowplow__derived_tstamp_partitioned: false
+```
+:::
+
```

Comment:
Do you think we should mention here the `snowplow__session_timestamp` variable to pick what your table is partitioned by, and recommend `load_tstamp` if it is partitioned by that (or on snowflake so micropartitioned)? Would make it more efficient for people?

Reply:
Shouldn't the `snowplow__derived_tstamp_partitioned` be obsolete and be inferred from the value the user gives for the `snowplow__session_timestamp`? I need to check what they all do.

Reply:
Sort of, I guess you could still want collector or load to be your incremental logic timestamp, but for BigQuery you _have_ to also filter on derived if it is your partition key. But yeah it would be easier for the user to set the session timestamp to their partition